release-notes/v1_69.md

```diff
@@ -203,6 +203,37 @@ There is now a Quick Access menu to view a list of your debug consoles! In the m
 
 _Theme: [GitHub Dark Dimmed](https://marketplace.visualstudio.com/items?itemName=GitHub.github-vscode-theme)_
 
+## Tasks
+
+### Decorations
+
+A few iterations ago, we added decorations to the terminal buffer and overview ruler to leverage shell integration capabilities and improve navigation of the terminal.
+
+Decorations now also mark points of interest for tasks and be jumped to with the command navigation feature (`ctrl/cmd+up`, `ctrl/cmd+down`)
+
+For watch tasks, a decoration appears by the first reported problem for each compilation.
+
+For start/stop tasks, a decoration appears next to the start of the task and is styled according to the run's exit code, if any.
+
+### Icon and color support
+
+An `icon` can now be set for a task with a Codicon `id` and a terminal ansi `color` property.
+
+For example:
+
+```
+{
+    "label": "test",
+    "type": "shell",
+    "command": "echo test",
+    "icon": { "id": "light-bulb", "color": "terminal.ansiBlue"}
+},
+
+```
+![The task icon appears in the terminal tabs list](./images/1_69/task-icon.png)
+
+Test tasks have the `beaker` icon by default.
+
 There are two ways to access this Quick Access menu:
 
```

Comment:
Bad merge in the middle of debug content

Reply:
yep fixing